README.md

```diff
@@ -2,23 +2,27 @@
 > expect(element).to.have.xpath('//blink')
 
 ## Installation
+```bash
+> npm install --save-dev chai-have-xpath
 ```
-npm install --save-dev chai-have-xpath
+
+## Development
+```bash
+> npm install
+> npm run watch
 ```
 
 ## Usage
 
 ```javascript
-var chai = require('chai');
-var React = require('react/addons');
-var TestUtils = React.addons.TestUtils;
-var chaiHaveXpath = require('chai-have-xpath');
-
+const chai = require('chai');
```

Comment:
I'll remove it, but NodeJS ships with block-level scoping enabled by default now